Reference Models Centrally, set OpenAI Model to gpt-5-nano and Updated Spec Defaults (#4608)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-conversation/openai.md

| Field              | Required | Details | Example |
|--------------------|:--------:|---------|---------|
| `key`   | Y | API key for OpenAI. | `mykey` |
| `model` | N | The OpenAI LLM to use. Defaults to `gpt-5-nano` (configurable via the `OPENAI_MODEL` environment variable).  | `gpt-5-nano` |
| `cacheTTL` | N | A time-to-live value for a prompt cache to expire. Uses Golang duration format.  | `10m` |

## Azure OpenAI usage

The `conversation.openai` component can target either OpenAI's hosted API or Azure OpenAI. To select Azure OpenAI, set the component's `apiType` metadata to `azure` and provide the usual Azure-specific connection settings (for example, endpoint/region and API key) in the component configuration.

> When `apiType: azure` is used, the environment variable `AZURE_OPENAI_MODEL` may be set to provide a default Azure model identifier to use when the component's `model` metadata is not provided. This environment variable only affects the component when `apiType` is set to `azure` — the regular `DAPR_CONVERSATION_OPENAI_MODEL` remains the default for non-Azure OpenAI usage.

Reviewer: ```suggestion
When `apiType: azure` is used, the environment variable `OPENAI_MODEL` may be set to provide a default Azure model identifier to use if the component's `model` metadata is not provided. In addition, the corresponding `endpoint` and `apiVersion` fields will be set to Azure specific settings.
```

Example (Azure OpenAI configuration):

```yaml
spec:
  type: conversation.openai
  metadata:
  - name: apiType
    value: azure
  - name: key
    value: "<your-azure-openai-key>"
  - name: endpoint
    value: "https://<your-resource-name>.openai.azure.com/"
  - name: model

>     value: '${{AZURE_OPENAI_MODEL}}'

Reviewer: ```suggestion
    value: 'gpt-4.1-nano'
```

```

> If `model` is omitted from the component metadata and neither `AZURE_OPENAI_MODEL` nor `DAPR_CONVERSATION_OPENAI_MODEL` are set, the component falls back to its built-in default model.

Reviewer: ```suggestion
If `model` is omitted from the component metadata and the model environment variable is not set, then the component falls back to its built-in default model.
```
